### ISSUE TITLE
ability to insert the new glb/fbx 3d format 

### ISSUE THREAD

**Issue by sriramab** [gama-platform/gama#2886]
Since Windows 10, users can **create their own 3D shapes**, and I think it would be very useful to be able to import these as agent aspects. 

The following 3d formats exports are now possible from windows 10 
1. `glb`
2. `fbx`
3. `3mf`

https://maps3d.io/blog/what-is-the-gltf-glb-format-3d-file/ 

Thank you.

**Comment by benoitgaudou** [comment]
And there is this lib : https://github.com/javagl/JglTF
(at least for glb)

**Comment by sriramab** [comment]
Yes, glb would be great, the default recommended is glb. I tried drawing a few 3d shapes and saving them on a touch laptop with a pen. Windows 10 makes it super easy to create such 3d objects. Would be great to have this feature.

**Comment by AlexisDrogoul** [comment]
@benoitgaudou using this library could actually be very interesting. I was not aware of this format and it could be quite handy as an intermediary format to represent remote displays in GL. Additional interesting libraries could be found here, including an obj parser that seems a bit more stable than ours: https://github.com/javagl?tab=repositories
